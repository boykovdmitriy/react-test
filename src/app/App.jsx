import React from "react";
import styles from "./app.style.css";

export default class App extends React.Component {
    render() {
        return (
            <section className={styles.panel}>Rendered</section>
        );
    }
}