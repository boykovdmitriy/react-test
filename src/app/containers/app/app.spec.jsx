import React from 'react';
import App from './index'

test('render App component', () => {
    const wrapper = shallow(
        <App/>
    );
    expect(wrapper).toMatchSnapshot();
});