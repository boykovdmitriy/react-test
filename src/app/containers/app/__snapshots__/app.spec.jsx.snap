// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render App component 1`] = `
<section
  className="panel"
>
  Rendered
</section>
`;
